cw/test_waverunner.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Test WaveRunner class."""
+
+from datetime import datetime
+from waverunner import WaveRunner
```

Comment:
```suggestion
from datetime import datetime

from waverunner import WaveRunner
```

I'm afraid that the remaining lint error message is not very informative.
You can run `isort -c -w79 test_waverunner.py --diff` locally to see what isort is expecting.
At the moment it is only complaining about the missing empty line.